src/airport.js

```diff
@@ -1,6 +1,44 @@
+const Plane = require(`./Plane.js`);
+
 class Airport {
-  // here's a starting point for you
 
-}
+    constructor(maximumCapacity = 10) {
+        this.airportPlanes = [];
+        this.maximumCapacity = maximumCapacity;
+    };
+
+    landPlane = plane => {
+        if (this.airportPlanes.length >= this.maximumCapacity) {
+            return this.airportPlanes.length;
```

Comment:
I know this should've been something like a boolean return instead of just returning the array size. I'm not sure why I chose to frame my tests around the size of the array from this return like this.